src/robusta/core/playbooks/prometheus_enrichment_utils.py

```diff
@@ -82,13 +87,23 @@ def create_chart_from_prometheus_query(
     else:
         chart.title = promql_query
     # fix a pygal bug which causes infinite loops due to rounding errors with floating points
+    min_time = None
+    max_time = None
     for series in result:
         label = "\n".join([v for v in series["metric"].values()])
         values = [
             (timestamp, round(float(val), FLOAT_PRECISION_LIMIT))
             for (timestamp, val) in series["values"]
         ]
+        times = [timestamp for (timestamp, _) in series["values"]]
+        if not min_time or min(times) < min_time:
```

Comment:
I would initialize min_time to big number, and max_time to 0
then:
min_time = min(min_time, min(times))
max_time = max(max_time, max(times))

no "ifs" is better imo